mycelo/loadbot/bot.go

```diff
@@ -40,42 +41,71 @@ func Start(ctx context.Context, cfg *Config) error {
 		return cfg.Accounts[idx].Address, cfg.Amount
 	}
 
-	for _, acc := range cfg.Accounts {
+	// developer accounts / TPS = duration in seconds.
+	// Need the fudger factor to get up a consistent TPS at the target.
+	delay := time.Duration(int(float64(len(cfg.Accounts)*1000/cfg.TransactionsPerSecond)*0.95)) * time.Millisecond
+	startDelay := delay / time.Duration(len(cfg.Accounts))
+
+	for i, acc := range cfg.Accounts {
+		// Spread out client load across different diallers
+		client := cfg.Clients[i%len(cfg.Clients)]
+
+		err := waitFor(ctx, startDelay)
+		if err != nil {
+			return err
+		}
 		acc := acc
 		group.Go(func() error {
-			client, err := cfg.ClientFactory()
-			if err != nil {
-				return err
-			}
-			return runBot(ctx, acc, cfg.TransactionDelay, client, nextTransfer)
+			return runBot(ctx, acc, cfg.Verbose, delay, client, nextTransfer)
 		})
 
 	}
 
 	return group.Wait()
 }
 
-func runBot(ctx context.Context, acc env.Account, sleepTime time.Duration, client bind.ContractBackend, nextTransfer func() (common.Address, *big.Int)) error {
+func runBot(ctx context.Context, acc env.Account, verbose bool, sleepTime time.Duration, client bind.ContractBackend, nextTransfer func() (common.Address, *big.Int)) error {
 	abi := contract.AbiFor("StableToken")
 	stableToken := bind.NewBoundContract(env.MustAddressFor("StableToken"), *abi, client)
 
 	transactor := bind.NewKeyedTransactor(acc.PrivateKey)
 	transactor.Context = ctx
 
+	stableTokenAddress := common.HexToAddress("0xd008")
```

Comment:
you should use `env.MustProxyAddressFor("StableToken")`. bring changes from master, line 69 is broken here